aries-site/src/pages/foundation/voice-and-tone.mdx

```diff
@@ -1 +1,361 @@
+- Document the type of message (alert, task, etc).
+- If the message doesn't document (for the user) full details of how this problem occurs or what to do about it, add the details in comments.
+- Some context around the task being performed or the resource category involved is useful.
+- Is the message an API-only message (does not appear in the UI)? If yes, document it as such.
+
+### Scale
+
+Consider singular and plural use cases and craft messages for each.
+
+If a sentence contains a list, do not allow the length of the list to be unbounded. Instead, consider a construct like this: *The following policies have not been implemented: Policy A, Policy B, Policy C, and 5 others.*
+
+### User Input
+
+Unless there is a compelling reason, do not limit the character set that a user can use to name things. Names commonly contain non-alphanumeric characters and we must handle them correctly on input and in re-display via messaging. For example:
+
+- *Joe’s system* (apostrophe for a possessive)
+- *O’Reilly Auto Parts* (apostrophe in name)
+- *Yahoo! Client* (company name includes a symbol)
+- *Frank James-Martin* (Hyphen in name)
+- *DC-NY-East* (Dashes in internal company identifiers)
+- *Björk Guðmundsdóttir* (Unicode characters)
+- *联想* (Unicode characters)
+- *John Q. Public* (punctuation for abbreviation)
+- *“”>* (An actual company name in the UK that the UK Companies House had to ban because it broke their systems. See [article](https://idle.slashdot.org/story/20/11/08/2334254/uk-agency-demands-company-stop-using-name-which-includes-an-html-closing-tag).)
+
+Proper encoding/decoding of user-entered input must be performed to prevent SQL/XSS attacks. Input such as the following user input must not cause problems when the user-generated input is later incorporated in application messaging:
+- *Robert’); DROP TABLE Students;--* (see: [https://xkcd.com/327/](https://xkcd.com/327/))
+
+User-generated input can be arbitrarily long. When confronted with displaying user-generated input, plan for the 80% case, but define which of truncation (with hover) or wrapping will be used.
+
+## Other Common Problems and How To Address Them
+ 
+The following are commonly misspelled/mis-capitalized terms:
+
+- The following must be spelled/capitalized as shown:
+  - Kubernetes
+  - Ethernet
+  - Fibre Channel
+  - FCoE
+  - SNMP, SNMPv3
+  - ID (not id or Id)
+  - OK (not ok, Ok, or okay)
+  - I/O (not IO or i/o)
+  - API (not xApi) (either that, or be explicit and say X-API-Version)
+- "Log in" is a verb (e.g. "Log in to the server").
+- "Login" is a noun or adjective (e.g. "The login token has expired").
+- Use the word "and" instead of an ampersand "&".
+- Filesystem is one word.
+- Use "host name" when referring to the name of a host.
+- Use "hostname" when referring to the Unix command.
+
+The following table contains a set of common mistakes found in messaging, along with suggested corrections:
+
+| Don't	| Do |Notes |
```

Comment:
More a comment for our team, but wondering if we should do some custom markdown styling to make these tables a bit more readable.